core/sql/regress/catman1/EXPECTED129

```diff
@@ -258,19 +258,16 @@ End of MXCI Session
 >>-- As user1, should succeed
 >>update us4.t1 set col3 = 2 ;
 
-*** ERROR[4481] The user does not have SELECT privilege on table or view TRAFODION.US4.T1.
-
-*** ERROR[8822] The statement was not prepared.
-
+--- 4 row(s) updated.
```

Comment:
This issue is documented in TRAFODION 1243. Thank you for fixing it. I am assigning this issue to you.